src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -45,63 +46,57 @@
 
 ;; ACE creation
 
-(defn munge-name
-  [pattern]
-  (-> pattern (str/lower-case) (str/split #"\.") reverse vec))
+(schema/defn ^:private split-domain :- [String]
+  "Given a domain string, split it on '.' and reverse it. For examples,
+  'sylvia.plath.net' becomes ['net' 'plath' 'sylvia']. This is used for domain
```

Comment:
without the `vec` call, this docstring should probably be updated to `('net' 'plath' 'sylvia')`